cf-java-logging-support-opentelemetry-agent-extension/README.md

```diff
@@ -1,13 +1,14 @@
 # OpenTelemetry Java Agent Extension for SAP Cloud Logging
```

Comment:
Title does not reflect the update. Either Dynatrace is missing or a more generic title should be used

Reply:
Will change it to SAP BTP Observability
```suggestion
# OpenTelemetry Java Agent Extension for SAP BTP Pbservability
```